src/main/webapp/script.js

```diff
@@ -634,15 +690,36 @@ function getTooltipContent(node) {
  * When a next/previous button is clicked, modifies the mutation index of the
```

Comment:
Also add some high-level documentation of what happens when the currMutationIndex is decimal

Reply:
I think it's in line 41! do you think we should move it somewhere else?

Reply:
line 41 talks about the fact that currMutationIndex could be decimal. What could be added here is what happens when currMutationIndex is decimal. e.g. "if currMutationIndex is decimal, next/prev would set it to the next/previous integer index"